django_mesh/templates/django_mesh/media_index_view.html

```diff
@@ -0,0 +1,23 @@
+{% extends "django_mesh/base.html" %}
+
+{% block content %}
+
+<div class="media_list">
+{% if media_list %}
+
+    {% for media in media_list %}
+        <div>
+            <a href="{% url 'mesh_media_view' slug=media.slug %}">{{ media.title|safe }}</a>
+            <div class="container2">
+
+            <p> <embed src="{{ media.file_url }}" autoplay="false" height="{{ media.thumbnail_height }}" width="{{ media.thumbnail_width }}">
```

Comment:
I think the embed tag is non-standard and doesn't work in all browsers. Typically \<embed> is placed inside an \<object> tag.

Does \<embed> work for non-videos? Was the intent to have the actual file in the index or just a thumbnail image?